pkgs/development/python-modules/icalendar-compatibility/default.nix

```diff
@@ -0,0 +1,49 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  hatchling,
+  hatch-vcs,
+  icalendar,
+  pytestCheckHook,
+  git,
+}:
+
+buildPythonPackage rec {
+  pname = "icalendar-compatibility";
+  version = "0.1.4";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "niccokunzmann";
+    repo = "icalendar_compatibility";
+    tag = "v${version}";
+    hash = "sha256-wb7iXj7iQDUiX3zsdtp64W7TYYHv9ZEdtJjIpvqK90M=";
+    # hatch-vcs tries to read the current git commit hash
```

Comment:
See https://github.com/NixOS/nixpkgs/pull/389323/files#diff-274b155667cdf3252cc069d8f2bce06081a8c66d6d610d333f70b56bed574080R30-R33 for how to deal with that.

Reply:
Good to know!